src/main/webapp/script.js

```diff
@@ -22,18 +22,36 @@ function loadComments() {
   });
 }
 
+/** Redirects the user to the result page, with the given parameter for user-query. */
+function redirectToResults(userQuery) {
+  document.location.href = "search-results-test.html?user-query=" + userQuery;
+  return false;
+}
+
+/** Gets the value of the given parameter from the current URL string. */
+function getURLParamVal(param) {
+  // Get the current page URL.
+  var pageURL = window.location.search.substring(1);
```

Comment:
Would something like URLSearchParams be preferred here so you don't need to be doing this string manipulation.

Reply:
Yes - fixed!